policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/vt_retirement_income_exemption.py

```diff
@@ -0,0 +1,61 @@
+from policyengine_us.model_api import *
+
+
+class vt_retirement_income_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Vermont retirement income exemption"
+    reference = (
+        "https://legislature.vermont.gov/statutes/section/32/151/05811",  # Titl. 32 V.S.A. § 5811(21)(B)(iv)
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-112%20Instr-2022.pdf#page=3",  # Instruction for 2022 SCHEDULE IN-112 - RETIREMENT INCOME EXEMPTION WORKSHEET
+    )
+    unit = USD
+    defined_for = StateCode.VT
+    documentation = "Vermont retirement benefits exempt from Vermont taxation."
+
+    def formula(tax_unit, period, parameters):
+        tax_unit_taxable_social_security = tax_unit(
+            "tax_unit_taxable_social_security", period
```

Comment:
you can deduct either social security or retirement income - need to check which deduction would be larger